src/main/java/com/google/cloud/teleport/templates/common/JavascriptTextTransformer.java

```diff
@@ -154,7 +158,8 @@ private static Invocable newInvocable(Collection<String> scripts) throws ScriptE
     public String invoke(String data) throws ScriptException, IOException, NoSuchMethodException {
       Invocable invocable = getInvocable();
       if (invocable == null) {
-        throw new RuntimeException("No udf was loaded");
+        LOG.error("No UDF was loaded");
```

Comment:
The log statement you added in the catch block below should be sufficient, no?

Reply:
Added it to log errors for other pipelines that use the `TransformTextViaJavascript` PTransform, which does not have a catch block. ([#L257](https://github.com/melbrodrigues/DataflowTemplates/blob/log-udf-errors/src/main/java/com/google/cloud/teleport/templates/common/JavascriptTextTransformer.java#L257))

Pub/Sub to Splunk uses the `FailsafeJavascriptUdf` PTransform for which the catch block would be sufficient.